.github/actions/install-dependencies/action.yml

```diff
@@ -18,18 +21,6 @@ runs:
           tsconfig.tsbuildinfo
         key: ${{ matrix.os }}-eslint-tsbuildinfo-${{ hashFiles('**/*.ts','**/*.js', 'package.json', 'tsconfig.json') }}
 
-    # Attempt to cache all the node_modules directories based on the OS and package lock.
-    - name: Cache node_modules
-      id: npm-cache
-      uses: actions/cache@v3
-      env:
-        cache-name: cache-node-modules
-      with:
-        key: ${{ runner.os }}-${{ env.node-version }}-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-        path: "**/node_modules"
-
-    # If the cache missed then install using `npm ci` to follow package lock exactly
-    - if: ${{ steps.npm-cache.outputs.cache-hit != 'true'}}
-      shell: bash
+    - shell: bash
```

Comment:
Do you know why we have this dotted throughout the actions? It seems to be used inconsistently. Is it even needed?

Reply:
I've removed from most places (left it in one place where I think it's needed to work on Windows)